pkg/reconciler/pipelinerun/pipelinerun.go

```diff
@@ -465,18 +476,33 @@ func (c *Reconciler) reconcile(ctx context.Context, pr *v1beta1.PipelineRun) err
 
 // runNextSchedulableTask gets the next schedulable Tasks from the dag based on the current
 // pipeline run state, and starts them
-func (c *Reconciler) runNextSchedulableTask(ctx context.Context, pr *v1beta1.PipelineRun, d *dag.Graph, pipelineState resources.PipelineRunState, as artifacts.ArtifactStorageInterface) error {
+// after all DAG tasks are done, its responsible for scheduling final tasks and start executing them
```

Comment:
NIT: I preferred the original comment with the extra line, but it's also ok

s/its/it's

Reply:
Done